<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="Btn_Voice">
   <property name="geometry">
    <rect>
     <x>694</x>
     <y>690</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/Icons/Volume.png</normaloff>:/img/Icons/Volume.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_OpenFile">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>4</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/Icons/folder.png</normaloff>:/img/Icons/folder.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="HS_SongProgress">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>694</y>
     <width>491</width>
     <height>21</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="CombBox_PlayMode">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>694</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Expand">
   <property name="geometry">
    <rect>
     <x>848</x>
     <y>0</y>
     <width>33</width>
     <height>29</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Icons/Expand.png</normaloff>Icons/Expand.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Min">
   <property name="geometry">
    <rect>
     <x>878</x>
     <y>0</y>
     <width>33</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Icons/min.png</normaloff>Icons/min.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_Close">
   <property name="geometry">
    <rect>
     <x>908</x>
     <y>0</y>
     <width>33</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Icons/close.png</normaloff>Icons/close.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Search">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>10</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_PlayList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>951</width>
     <height>631</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_PreMusic">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>690</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/Icons/Previous.png</normaloff>:/img/Icons/Previous.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_NextMusic">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>690</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/Icons/Next.png</normaloff>:/img/Icons/Next.png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_PlayPause">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>690</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/img/Icons/Play.png</normaloff>:/img/Icons/Play.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="HS_Volume">
   <property name="geometry">
    <rect>
     <x>734</x>
     <y>695</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl_SongName">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>670</y>
     <width>501</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Lbl_ShowTime">
   <property name="geometry">
    <rect>
     <x>565</x>
     <y>708</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ShowTime:</string>
   </property>
  </widget>
  <zorder>Btn_Voice</zorder>
  <zorder>Btn_OpenFile</zorder>
  <zorder>HS_SongProgress</zorder>
  <zorder>CombBox_PlayMode</zorder>
  <zorder>Btn_Expand</zorder>
  <zorder>Btn_Min</zorder>
  <zorder>Btn_Close</zorder>
  <zorder>lineEdit_Search</zorder>
  <zorder>tableView_PlayList</zorder>
  <zorder>Btn_PreMusic</zorder>
  <zorder>Btn_NextMusic</zorder>
  <zorder>Btn_PlayPause</zorder>
  <zorder>HS_Volume</zorder>
  <zorder>Lbl_SongName</zorder>
  <zorder>Lbl_ShowTime</zorder>
  <zorder>Lbl_SongName</zorder>
  <zorder>Lbl_ShowTime</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
